reflex/components/core/upload.py

```diff
@@ -293,20 +293,23 @@ def create(cls, *children, **props) -> Component:
             format.to_camel_case(key): value for key, value in upload_props.items()
         }
 
-        use_dropzone_arguments = {
-            "onDrop": event_var,
-            **upload_props,
-        }
+        use_dropzone_arguments = Var.create(
+            {
+                "onDrop": event_var,
+                **upload_props,
+            }
+        )
 
         left_side = f"const {{getRootProps: {root_props_unique_name}, getInputProps: {input_props_unique_name}}} "
-        right_side = f"useDropzone({str(Var.create(use_dropzone_arguments))})"
+        right_side = f"useDropzone({str(use_dropzone_arguments)})"
```

Comment:
I think f-string have an implicit call to str() by default, no ?

Reply:
iirc `__format__` takes precedence over `__str__`

Reply:
> We can use the f-string flag `!s`

Interesting, i didn't realize that would bypass the normal `__format__` logic, but makes sense.

I think it's a bit cleaner looking, however I'm hesitant to make the change in this PR because we have the `f"{str(x)}"` pattern spread throughout the codebase already. If we're going to use `!s`, then we should have a separate PR that does it everywhere.

Reply:
I mentioned this because I actually had a local PR where I'm doing this specific changes, let's go with this and I'll do the change on my PR.